rdr_service/workflow_management/nph/sms_workflows.py

```diff
@@ -146,6 +147,23 @@ def job_ingestion(self):
         Main method for ingestion jobs.
         """
 
+        file_end = self.file_path.split(".")[-1]
+        # Ensure file is CSV
+        if not file_end == "csv":
```

Comment:
Maybe a good idea to cast this a lower() and do the equivalence check then.
Also a  better variable could be helpful here like `file_extension`.